tessellation/src/path_stroke.rs

```diff
@@ -116,6 +116,18 @@ impl StrokeTessellator {
     }
 }
 
+macro_rules! add_vertex {
+    ($builder: expr, $vertex: expr) => {{
+        let mut v = $vertex;
+
+        if $builder.options.apply_line_width {
+            v.position += v.normal * $builder.options.line_width;
```

Comment:
We should extrude the position by half of the line width on each side so that the total width of the curve is equal to `line_width`.